configator/generate_readme.py

```diff
@@ -0,0 +1,11 @@
+"""Element of the Configator tool to enable default README.md generation"""
+
+print("Generating default README...!")
+
+def create_readme():
+    """ method to generate README
+    """
+    with open("../README.md", "w", encoding="utf8") as read_me:
+        read_me.write("This is text.")
+
+create_readme()
```

Comment:
This function should be called in the main program, not in this module. Otherwise, the README will be generated anytime this module is imported, which might not be the desired behavior.

This should either be removed or nested in an `if __name__ == '__main__':` statement (preferably the former) as to not decrease the test coverage.